testhelpers/core.go

```diff
@@ -113,11 +113,26 @@ func VMStorage() vm.StorageMap {
 	return vm.NewStorageMap(blockstore.NewBlockstore(datastore.NewMapDatastore()))
 }
 
-// CreateTestMiner creates a new test miner with the given peerID and miner
+// MustSign signs a given address with the provided mocksigner or panics if it
+// cannot.
+func MustSign(s types.MockSigner, msgs ...*types.Message) []*types.SignedMessage {
```

Comment:
Why is this being added? It doesn't seem to be used in this PR.

Reply:
I did not see that; it must have been pulled in when I did a rebase.